plugins/woocommerce/changelog/43274-add-rebase-42081-additional-fields

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: add
+Comment: Allow additional fields to be rendered at the end of the Checkout block.  This is locked behind a feature flag, the fields won't show up until we remove the feature gate.
```

Comment:
So we can't skip changelog?

Reply:
This should skip, since it's a comment, not a message 👌🏼